docs/ACCESSIBILITY.md

```diff
@@ -293,6 +293,46 @@ To make the live region more consistent across browsers and easier to control, w
    -  Only change focus synchronous to user gesture
    -  Related to [#3135](https://github.com/microsoft/BotFramework-WebChat/issues/3135)
 
+# Controlling the narration of activities and attachments
+
+> This is related to [#3360](https://github.com/microsoft/BotFramework-WebChat/issues/3360), [#3615](https://github.com/microsoft/BotFramework-WebChat/issues/3615), [#3918](https://github.com/microsoft/BotFramework-WebChat/issues/3918).
+
+## User story
+
+A bot developer want to set the narration for a message activity. The activity may or may not have attachments.
```

Comment:
A bot developer wants to set the narration for a message activity. The activity may or may not have attachments.